fix(a11y): final fix for WCAG 1.1.1 image alt should describe image content accurately (#17)
scottrigby/basekamp-next: content/projects/connect-dots.md

attribution: Scott Rigby & David Dempewolf
images:
  - src: DSC09299.jpg
    alt: Gallery view with visitors examining colorful science fair-style tri-fold display boards arranged on tables
  - src: DSC09280.jpg
    alt: Installation view of multiple colored tri-fold display boards in white gallery space with colorful wall panels in background
  - src: DSC09281.jpg
    alt: Elevated view of exhibition showing numerous tri-fold display boards on tables with colorful rectangular panels mounted on walls
  - src: DSC09283.jpg
    alt: Two visitors viewing science fair-style exhibition displays in white gallery space
  - src: DSC09284.jpg
    alt: Person working at laptop with wall diagram of colored squares connected by black lines visible behind them
  - src: DSC09285.jpg
    alt: Gallery view with visitors examining various colored tri-fold display boards during exhibition opening
  - src: DSC09288.jpg
    alt: Exterior view of Columbia University campus with Low Memorial Library's neoclassical columned facade
  - src: DSC09289.jpg
    alt: Visitors gathered around tri-fold display boards examining curatorial project presentations
  - src: DSC09290.jpg

>     alt: Gallery opening with visitors viewing exhibition displays, "con" visible on wall, colorful paintings in background

Reviewer: ```suggestion
    alt: Gallery opening with visitors viewing colorful tri-fold display boards, with wall installation of dots and connecting lines visible in the background
```

  - src: DSC09301.jpg
    alt: Group of visitors examining tri-fold display with collage of photographs during opening reception
  - src: DSC09302.jpg
    alt: Visitors viewing colorful tri-fold displays during opening event, "con_nect" visible on wall in background
  - src: DSC09303.jpg
    alt: Visitors examining orange and yellow tri-fold display boards with "Popular Archive" label visible
  - src: DSC09304.jpg
    alt: Close-up view of yellow and orange tri-fold display boards showing detailed documentation during exhibition
  - src: DSC09305.jpg
    alt: Visitors during opening reception viewing tri-fold display boards with "con_nect" text visible on wall
  - src: DSC09306.jpg
    alt: Visitors examining orange "Popular Archive" display board with "connect" wall text visible
  - src: DSC09307.jpg
    alt: Yellow tri-fold display showing "Ongoing Media Resources Monitoring" with photographs, orange display visible alongside
  - src: DSC09308.jpg
    alt: Gallery view with visitors examining display boards during opening, colorful posters and wall panels visible
  - src: DSC09309.jpg

>     alt: Visitor examining green and yellow display boards with colorful graphic prints mounted on wall behind

Reviewer: ```suggestion
    alt: Person standing next to colorful display boards with text 'connect' visible on the wall behind them
```

  - src: DSC09310.jpg

>     alt: Visitor reading yellow display with wall diagram of network connections using colored squares and lines behind them

Reviewer: ```suggestion
    alt: Visitors standing next to wall diagram of network connections using colored squares and lines behind them
```

  - src: DSC09312.jpg

>     alt: Man reading "Locus" display board while visitors converse in background near "con_nect" wall text

Reviewer: ```suggestion
    alt: View of the backs of the display boards creating a white corridor on long white tables, gallery visitors in the background
```

  - src: DSC09313.jpg
    alt: Three men in conversation at exhibition opening near display boards and "con_nect" wall text
  - src: DSC09315.jpg

>     alt: Visitors viewing large wall diagram showing network of colored squares connected by black lines

Reviewer: ```suggestion
    alt: Visitors at exhibition among colorful tri-fold display boards
```

  - src: card_front.jpg

>     alt: Network diagram showing all 60 participant names connected by colored lines and squares radiating from central "1000000 mph" hub

Reviewer: ```suggestion
    alt: Network diagram showing all 60 participant names connected by colored lines and squares
```
